src/groupeddataframe/splitapplycombine.jl

```diff
@@ -598,13 +460,20 @@ function combine(gd::GroupedDataFrame, p::Pair; keepkeys::Bool=true)
     else
         cs = p_from
     end
-    return combine_helper(cs => p_to, gd, keepkeys=keepkeys)
+    return combine_helper(cs => p_to, gd, keepkeys=keepkeys, regroup=regroup,
+                          copycols=true, keeprows=false)
 end
 
-function combine(gd::GroupedDataFrame,
-                 @nospecialize(cs::Union{Pair, typeof(nrow),
-                                         ColumnIndex, MultiColumnIndex}...);
-                 keepkeys::Bool=true)
+combine(gd::GroupedDataFrame,
+        cs::Union{Pair, typeof(nrow), ColumnIndex, MultiColumnIndex}...;
+        keepkeys::Bool=true, regroup::Bool=false) =
+    _combine_executor(gd, cs..., keepkeys=keepkeys, regroup=regroup,
+                      copycols=true, keeprows=false)
+
+function _combine_executor(gd::GroupedDataFrame,
```

Comment:
The main definition I can find of "executor" is "a person who is named to carry out the terms of a will." :-D
Maybe use a more joyful term?

Reply:
Right - changed to `_combine_prepare`. Not sure it is best though. The idea is that we need an "intermediate" function as:
* `select` calls `_combine_prepare` which later calls `_combine_helper`
* `combine` calls `_combine_prepare` (which later calls `_combine_helper`) if its first argument is `GroupedDataFrame`
* `combine` directly calls `_combine_helper` if its first argument is a `Pair` or a `Callable`